rasa/core/channels/hangouts.py

```diff
@@ -207,8 +206,19 @@ def __init__(
         self.hangouts_room_added_intent_name = hangouts_room_added_intent_name
         self.hangouts_user_added_intent_name = hangouts_removed_intent_name
 
+        # Google's Request obj (this is used to make HTTP requests) and cached
```

Comment:
I would rewrite this comment a bit since it was confusing at first.
```suggestion
        # Google's Request obj (this is used to make HTTP requests) is stored as an attribute with the cached
```